management/src/test/java/org/ehcache/management/cluster/ClusteringManagementServiceTest.java

```diff
@@ -234,15 +235,22 @@ private void remotelyUpdateCollectedStatistics() throws Exception {
       assertThat(client, is(notNullValue()));
       final ClientIdentifier ehcacheClientIdentifier = client;
 
+      CountDownLatch callCompleted = new CountDownLatch(1);
       AtomicReference<String> managementCallId = new AtomicReference<>();
       BlockingQueue<ContextualReturn<?>> returns = new LinkedBlockingQueue<>();
 
       agent.setContextualReturnListener(new ContextualReturnListener() {
         @Override
         public void onContextualReturn(ClientIdentifier from, String id, ContextualReturn<?> aReturn) {
-          assertEquals(ehcacheClientIdentifier, from);
-          assertEquals(managementCallId.get(), id);
-          returns.offer(aReturn);
+          try {
```

Comment:
I can't say that for this test I expect the exception to be thrown, and if it does, something went really wrong with this test

I'm using the latch here just to make sure the call returned, before doing any kind of assertion in the response (in some executions, the response was received before the request "call" returned)